### PR TITLE
zh-translation: docs/root/intro/arch_overview/operations/runtime.rst

### PR DESCRIPTION
**Pull request by scilla0531**
ref: https://github.com/cloudnativeto/envoy/issues/70

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/operations/runtime.rst [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-Envoy supports “runtime” configuration (also known as "feature flags" and "decider"). Configuration
-settings can be altered that will affect operation without needing to restart Envoy or change the
-primary configuration. The currently supported implementation uses a tree of file system files.
-Envoy watches for a symbolic link swap in a configured directory and reloads the tree when that
-happens. This type of system is very commonly deployed in large distributed systems. Other
-implementations would not be difficult to implement. Supported runtime configuration settings are
-documented in the relevant sections of the operations guide. Envoy will operate correctly with
-default runtime values and a “null” provider so it is not required that such a system exists to run
-Envoy.
+Envoy 支持“运行时”配置（也称为“功能标志”和“决策器”）。可以通过更改配置影响操作，而无需重新启动 Envoy 或更改主要配置。当前支持的实现使用文件系统文件树。Envoy 会观察配置目录中的符号链接交换，并在发生这种情况时重新加载树。这种类型的系统通常部署在大型分布式系统中。其他实现也不难实现。支持的运行时配置设置在操作指南的相关章节中进行了说明。Envoy 将在默认的运行时值和“null”提供程序的情况正常运行，因此不需要存在这样的系统即可运行 Envoy。
```
Comment (helight) [inline docs/root/intro/arch_overview/operations/runtime.rst]: 适当加个换行吧

**Review by scilla0531 — COMMENTED** [review]
运行时 :ref:`配置 <config_runtime>`。 //haha，中文标点

File: docs/root/intro/arch_overview/operations/runtime.rst (hunk 1 above)
Comment (scilla0531) [reply docs/root/intro/arch_overview/operations/runtime.rst]: 嗯嗯，好的。谢谢！

File: docs/root/intro/arch_overview/operations/runtime.rst [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
-settings can be altered that will affect operation without needing to restart Envoy or change the
-primary configuration. The currently supported implementation uses a tree of file system files.
-Envoy watches for a symbolic link swap in a configured directory and reloads the tree when that
-happens. This type of system is very commonly deployed in large distributed systems. Other
-implementations would not be difficult to implement. Supported runtime configuration settings are
-documented in the relevant sections of the operations guide. Envoy will operate correctly with
-default runtime values and a “null” provider so it is not required that such a system exists to run
-Envoy.
+Envoy 支持“运行时”配置（也称为“功能标志”和“决策器”）。可以通过更改配置影响操作，而无需重新启动 Envoy 或更改主要配置。
+当前支持的实现使用文件系统文件树。Envoy 会观察配置目录中的符号链接交换，并在发生这种情况时重新加载树。这种类型的系统通常部署在大型分布式系统中。其他实现也不难实现。
```
Comment (helight) [inline docs/root/intro/arch_overview/operations/runtime.rst]:
```suggestion
当前支持的实现使用文件系统文件树。Envoy 会观察配置目录中的符号链接交换，并在发生这种情况时重新加载树。
这种类型的系统通常部署在大型分布式系统中。用其它方式实现也不难。
```

File: docs/root/intro/arch_overview/operations/runtime.rst [hunk 3]
```diff
@@ -1,16 +1,10 @@
...
+支持的运行时配置设置在操作指南的相关章节中进行了说明。Envoy 将在默认的运行时值和“null”提供程序的情况正常运行，因此不需要存在这样的系统即可运行 Envoy。
```
Comment (helight) [inline docs/root/intro/arch_overview/operations/runtime.rst]:
```suggestion
支持的运行时配置的相关设置在操作指南的相关章节中进行了说明。Envoy 能使用默认的运行时值和“null”提供程序正常运行，因此不需要这样的系统也可以运行 Envoy。
```

**Pull request merged**